appconfig/torqv5process.csv

```diff
@@ -28,4 +28,4 @@ localhost,{KDBBASEPORT}+26,tailer_seg1,tailer1,${TORQAPPHOME}/appconfig/password
 localhost,{KDBBASEPORT}+27,tailer_seg2,tailer2,${TORQAPPHOME}/appconfig/passwords/accesslist.txt,1,1,,,${KDBCODE}/processes/tailer.q,1,-segid 2 -parentproctype tailer,q
 localhost,{KDBBASEPORT}+28,tr_seg1,tr1,${TORQAPPHOME}/appconfig/passwords/accesslist.txt,1,1,,,${KDBCODE}/processes/tailreader.q,1,-segid 1 -parentproctype tailreader,q
 localhost,{KDBBASEPORT}+29,tr_seg2,tr2,${TORQAPPHOME}/appconfig/passwords/accesslist.txt,1,1,,,${KDBCODE}/processes/tailreader.q,1,-segid 2 -parentproctype tailreader,q
-
+localhost,{KDBBASEPORT}+30,eodprocess,eod1,${TORQAPPHOME}/appconfig/passwords/accesslist.txt,1,1,,,${KDBCODE}/processes/eodprocess.q,1,,q
```

Comment:
Was thinking that maybe eodprocess is a bit misleading, could name it something along the lines of cloudsortprocess (not the best name and probably not what I would go with but puts more emphasis on sorting).

Reply:
How about cloudsort or eodsort (since we have sort above for classic TorQ)?

Reply:
perhaps `tailsort` or `tailersort`? TorQ v5 doesn't _have_ to be used on cloud, and the other sort process also occurs at eod, so I think either of those could be confusing or misleading, explicitly linking it to the tailer might be more clear